File: src/main/java/com/dipol/pdfconvert/data/entity/Da01.java
package com.dipol.pdfconvert.data.entity;

import lombok.Data;

import javax.persistence.*;

import java.util.Date;

@Entity
@Data
public class Da01 {
    @Id
    @Column(name = "acc_no")
    private String accNo;
    @Column(name = "prod_name")
    private String prodName;
    @Column(name = "car_desc")
    private String carDesc;
    @Column(name = "vin")
    private String vin;
    @Column(name = "user_name")
    private String userName;
    @Column(name = "write_date")
    private String writeDate;
    @Column(name = "tool_version")
    private String toolVersion;
    @Column(name = "report_version")
    private String reportVersion;
    @Column(name = "status_flag")
    private String statusFlag;
    @Column(name = "doc_path")
    private String docPath;
    @Column(name = "add_user")
    private String addUser;
    @Column(name = "add_date")
    private Date addDate;
    @Column(name = "add_ip")
    private String addIp;
    @Column(name = "update_user")
    private String updateUser;
    @Column(name = "update_date")
    private Date updateDate;
    @Column(name = "update_ip")
    private String updateIp;
    @Column(name = "bigo")
    private String bigo;
}

File: vue/src/store/da01.ts
import { defineStore } from 'pinia'

import {deleteBa04, getDa01List, Page, writeDa01} from "../api";
import da01Write from "../views/Da01Write.vue";

export interface da01Type {
    accNo: String
    prodName: String
    carDesc: String
    vin: String
    userName: String
    writeDate: String
    toolVersion: String
    reportVersion: String
    statusFlag: String
    docPath: String
    addUser: String
    addDate?: Date
    addIp?: String
    updateUser?: String
    updateDate?: Date
    updateIp?: String
    bigo: String,
    pdfFile?:File

}
export const Da01Store = defineStore("Da01Store",
    {
        state: () => ({
            da01List:[] as da01Type[],
        }),
        getters: {


        },
        actions: {
            getDa01List(params : Page) {
                return getDa01List(params)
            },
            delete(params : String) {
                return null
            },
            write(params:da01Type) {
                return writeDa01(params)
            },
            getBean() {
                const da01 : da01Type={
                    accNo: '',
                    prodName: '',
                    carDesc: '',
                    vin: '',
                    userName: '',
                    writeDate: '',
                    toolVersion: '',
                    reportVersion: '',
                    statusFlag: '',
                    docPath:'',
                    addUser: '',
                    bigo: '',
                }
                return da01
            }

        }
    })